sw/host/tests/chip/i2c_target/src/main.rs

```diff
@@ -142,17 +142,14 @@ fn test_write_repeated_start(
     _opts: &Opts,
     transport: &TransportWrapper,
     address: u8,
+    gpio_pins: &[Rc<dyn GpioPin>],
 ) -> Result<()> {
     let uart = transport.uart("console")?;
 
     log::info!("Emulating start with bit-banging");
     let gpio_bitbanging = transport.gpio_bitbanging()?;
```

Comment:
No need to change it here, but FYI, the hyperdebug firmware should be capable of handling repeated starts now. The SMBus ARP test uses it and checks that it works. :)